### PR TITLE
Added Splash screen

### PR DESCRIPTION
**Pull request by kikettas**
Added splash screen and app coordinator to the app.

### How to test
Launch the app.

### Merging
Automatic.

### Ticket 
Issue #2

### CODE REVIEW

**Comment by kikettas** [comment]
Despite there is some commits after the PR creation, I just rebase the branch with develop so we avoid potential conflicts.

File: GiniVisionExample/GiniVisionExampleTests/GiniVisionExampleTests-Bridging-Header.h [hunk 1]
```diff
@@ -0,0 +1,4 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
```
Comment (nsobadjiev) [inline GiniVisionExample/GiniVisionExampleTests/GiniVisionExampleTests-Bridging-Header.h]: Why did you add a Bridging header to the project? We don't use Objective-C.

**Review by nsobadjiev — APPROVED** [review]
I don't like that there seems to be several items in this pull request - switching to app coordinator, removing the storyboard and modifying the splash screen.
I also don't understand they Bridging header.

Other than that - looks good.

File: GiniVisionExample/GiniVisionExampleTests/GiniVisionExampleTests-Bridging-Header.h (hunk 1 above)
Comment (kikettas) [reply GiniVisionExample/GiniVisionExampleTests/GiniVisionExampleTests-Bridging-Header.h]: I thought the same, but when I created the test for the first time Xcode asked to have one but we can remove it yep
